starlette/testclient.py

```diff
@@ -259,13 +268,15 @@ def __init__(
         self.headers.update({"user-agent": "testclient"})
         self.base_url = base_url
 
-    def request(self, method: str, url: str, **kwargs) -> requests.Response:
+    def request(  # type: ignore
+        self, method: str, url: str, **kwargs: typing.Any
```

Comment:
I guess we *could* drop `**kwargs` and instead follow the complete `requests` signature here, as per https://github.com/requests/requests/blob/master/requests/sessions.py#L457

Though it might end up being a bit of work to figure all the correct annotations to use? Any thoughts?